src/pages/Main/Main.component.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Home from '../Home/index';
+import VideoDetail from '../../components/VideoDetail/VideoDetail';
+import { Route, Switch } from 'react-router-dom';
+import { useVideoContext } from '../../state/VideosContext';
+import { storage } from '../../utils/storage';
+import Favorites from '../../components/Favorites/Favorites';
+import { USER_FAVORITE_VIDEOS } from '../../utils/constants';
+
+function MainPage(){
+    const { videoList } = useVideoContext();
```

Comment:
I saw that in some components you are passing to their childs the videoList, if we already have in the context it would be better to read from there instead of passing it